### ISSUE TITLE
Propose to have a diagram which can visualize all the connection between different Eiffel events types

### ISSUE THREAD

**Issue by kennethlwk** [eiffel-community/eiffel#281]
### Description
Propose to have a diagram which can visualize all the connection between different Eiffel events types.

Eiffel events are interconnected, there are some rules specified in “Links” event in Eiffel documentation, but it is still hard to get the whole picture on which event type can be the input and output of that event type by just looking on the "Links" field of every single event. 

### Motivation
It is really difficult to know the connection between different event types.

### Exemplification
For example, in the Links field in CLM event, it has specified different link types(SUBJECT, SUB_CONFIDENCE_LEVEL, CAUSE, CONTEXY, FLOW_CONTEXT), and under each type it specified “Legal targets” eventType. The connection in the link of all the events should have been shown in a diagram.

### Benefits
Help user to understand and follow the rule of Eiffel events chain, and which will eventually produce more quality Eiffel data.

### Possible Drawbacks
No

**Comment by m-linner-ericsson** [comment]
An overview picture with all events and connections can be quite cumbersome to read.
![eiffel-vocabulary-master-2022-06-01-(post-Lyon) pa2](https://user-images.githubusercontent.com/54929896/177312422-b943090b-8410-4373-98f8-9f9fa38203d8.svg)

We probably need smaller chunks of this picture to make it more readable. For example looking at only `ArtC` 
![image](https://user-images.githubusercontent.com/54929896/177312629-01671d8e-3f82-4f8d-a99f-a482e938f1b4.png)
it becomes easier to make something out of it.

**Comment by magnusbaeck** [comment]
Yes. Something like the second variant on each event page (eiffel-vocabulary/EiffelMumbleEvent.md) centered on that event type  with all inbound and outbound links would be very useful. Ideally it should be an SVG where the other event type boxes could be hyperlinks.
